docs/assets/js/main.js

```diff
@@ -371,3 +371,41 @@ window.addEventListener('hashchange', () => {
         behavior: 'smooth',
     });
 });
+
+const makeTwoPartCallback = () => {
```

Comment:
@nkdengineer I think we should add a comment explaining why we need this function because it is not immediately clear what `makeTwoPartCallback` means.

what do you think?